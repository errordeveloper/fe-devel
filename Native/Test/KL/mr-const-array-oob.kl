operator entry() {
  Integer va[];
  for (Integer i=0; i<10; ++i)
    va.push(i);
  ArrayProducer<Integer> ap = createConstArray(va);
  // should cause exception
  report ap.produce(11);
}
