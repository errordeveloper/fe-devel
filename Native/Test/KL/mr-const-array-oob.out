Caught exception: index out of range
